@xen-orchestra/lite/src/locales/en.json

```diff
@@ -74,6 +74,7 @@
   "new-features-are-coming": "New features are coming soon!",
   "object": "Object",
   "object-not-found": "Object {id} can't be found…",
+  "open-in-new-window": "Open in new window",
```

Comment:
IMO, `Open in a new tab` will be more correct because we don't open a new window